CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Changelog
 
-## 2.0-alpha
+## 2.0.0-beta
+  - Added backwards compatibility for
+    - All the navigation methods
+    - ScreenLifecycleListener
+    - History re-writer
+    - NavigationType
+  - Added ability to turn off animations and logging
+  - Added ability to provide default transitions
+  - Breaking changes:
+    - Removed action bar related code
+    - Removed activity reference from screens
```

Comment:
Oooh we got rid of this for existing screens too? Or just Steps?

Reply:
Both!